### ISSUE TITLE
Docs for Katello 4.1/4.3 are missing

### ISSUE THREAD

**Issue by maximiliankolb** [theforeman/theforeman.org#1961]
Hi everyone.
I was trying to view Katello documentation on SCA for version 4.1 and 4.3:

<img width="768" alt="Screenshot 2022-02-23 at 10 41 05" src="https://user-images.githubusercontent.com/12595287/155294403-f62e12c7-4646-4613-afb4-2f6a8e233ef3.png">

Is this intentional? Is the Katello documentation in the Foreman manual deprecated?

**Comment by ekohl** [comment]
Yes, since Katello 4.0 the docs.theforeman.org site is the official documentation. I've been trying to make this clear by repeating it multiple times but clearly this has failed. @evgeni also raised it recently that that particular index site you opened needs to show this.

**Comment by maximiliankolb** [comment]
That's actually great to hear. One suggestion: drop the "nightly" for Katello then, and instead refer users to docs.theforeman.org.

To me, _nightly_ suggests that it's up to date.

**Comment by ekohl** [comment]
Last item on https://community.theforeman.org/t/release-team-meeting-agenda-2022-02-21/27429/2. I'm not sure we actually have someone assigned to it now so feel free to come with a proposal.

**Comment by lzap** [comment]
A landing page on the old site with link to new location would help I suppose.
